app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt

```diff
@@ -2,11 +2,19 @@ package com.theopensourcefamily.chessclock
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import com.jakewharton.rxbinding3.view.clicks
+import io.reactivex.Observable
 import kotlinx.android.synthetic.main.activity_clocks.*
 
-class ClocksActivity : AppCompatActivity(), GameView {
+class ClocksActivity : AppCompatActivity(), ClocksView {
 
-  val presenter = GamePresenter() // maybe DI here, with koin or wathever
+  val presenter = ClocksPresenter() // maybe DI here, with koin or wathever
```

Comment:
Great idea! What do you think about adding koin here or in another PR, I don't care.